src/test/resources/com/puppycrawl/tools/checkstyle/grammar/java14/InputJava14Records.txt

```diff
@@ -214,8 +214,6 @@ CLASS_DEF -> CLASS_DEF [19:0]
     |   |   `--LITERAL_PUBLIC -> public [39:4]
     |   |--LITERAL_RECORD -> record [39:11]
     |   |--IDENT -> Polar [39:18]
-    |   |--IMPLEMENTS_CLAUSE -> implements [39:48]
-    |   |   `--IDENT -> Coords [39:59]
     |   |--LPAREN -> ( [39:23]
```

Comment:
Here we can see why this needed to be fixed anyway; the `IMPLEMENTS_CLAUSE` on line 217 has a column number greater than the `LPAREN` on line 219.